client/src/components/RenderReviewList.tsx

```diff
@@ -0,0 +1,209 @@
+        compact: true,
+    },
+    {
+        name: 'Pull Request',
+        format: (row: CodeReview) => <a href={row.pullRequestLink}>{row.pullRequestLink.replace(/.*\/(.*?\/pull\/\d+)/,'$1')}</a>,
+        selector: () => true,
+        maxWidth: '20em',
+        compact: true,
+    },
+    {
+        name: 'Slack Link',
+        format: (row: CodeReview) => row.slackThreadTs && row.slackPermalink && <a href={row.slackPermalink ?? '#'}>{row.slackThreadTs}</a>,
+        selector: () => true,
+        maxWidth: '12em',
+        compact: true,
+    },
+    {
+        name: 'Reviewers',
+        selector: (row: CodeReview) => row.reviewers?.join(', ') ?? '',
+        wrap: true,
+        compact: true,
+    },
+    {
+        name: 'Approvers',
+        selector: (row: CodeReview) => row.approvers?.join(', ') ?? '',
+        wrap: true,
+        compact: true,
+    },
+    {
+        name: 'Request Changes',
+        selector: (row: CodeReview) => row.requestChanges?.join(', ') ?? '',
+        wrap: true,
+        compact: true,
+    },
+];
+
+// @seet https://react-data-table-component.netlify.app/?path=/docs/pagination-options--options
+const paginationComponentOptions = {
+    rowsPerPageText: 'Rows per page',
+    rangeSeparatorText: 'of',
+    selectAllRowsItem: false,
+};
+
+const customStyles = {
+    header: {
+        style: {
+            backgroundColor: '#555555',
+            color: '#eeeeee',
+        },
+    },
+    headRow: {
+        style: {
+            backgroundColor: '#505050',
+            color: 'white',
+            fontSize: '1.1em',
+        },
+    },
+    expanderRow: {
+        style: {
+            backgroundColor: '#252525',
+        }
+    }
+};
+
+createTheme('custom', {
+    text: {
+        primary: '#c0c0c0',
+        secondary: '#c0c0c0',
+    },
+    background: {
+        default: '#303030',
+    },
+    divider: {
+        default: '#505050',
+    },
+    striped: {
+        default: '#353535',
+    },
+    highlightOnHover: {
+        default: '#404040',
+    }
+}, 'dark');
+
+const RenderReviewList = ({channel, status, user}: {channel: string; status: string; user: {displayName: string}}) => {
+    const [dataSet, setDataSet] = useState([] as CodeReview[]);
+    const [loading, setLoading] = useState(true);
+    const [totalRows, setTotalRows] = useState(0);
+    const [pageSize, setPageSize] = useState(10);
+    const [currentPage, setCurrentPage] = useState(1);
+
+    /**
+     * Fetch the list of code review.
+     *
+     * @param {string} channel
+     * @param {number} limit
+     * @param {number} page
+     * @param {string} status
+     * @returns void
+     */
+    const updateFilter = ({channel, limit, page, status}: {channel: string; limit: number; page: number; status: string}) => {
+        setLoading(true);
+        fetchData(`/api/reviews/${channel}/${status}?limit=${limit}&page=${page}`)
+            .then((result) => {
+                if (!result) {
+                    return;
```

Comment:
We should probably set loading to false here as well `setLoading(false);`

Reply:
yes, good catch